Add release-drafter GitHub Action workflow (#237)
open-component-model/ocm: .github/pull_request_template.md

**Special notes for your reviewer**:

**Release Notes**:
<!--

> Please ensure that the title of this PR is good so that it can be added to the release notes.

Reviewer: ```suggestion
Please ensure that the title of this PR is suitable for inclusion in the release notes.
```
